riptide-spring-boot-autoconfigure/README.md

```diff
@@ -27,11 +27,23 @@ riptide.clients:
       enabled: true
       fixed-delay: 50 milliseconds
       max-retries: 5
+      threads:
+        max-size: 10
+        min-size: 2
+        enabled: true
```

Comment:
If I set this to `false` (as a user), I'd expect failsafe to use the calling thread here (i.e. no custom thread pool at all) - but actually it would use the (default) fork-join-pool then, right? This might be a bit confusing for users.

Reply:
I believe this was earlier discussed and therefore it was added to the README https://github.com/zalando/riptide/tree/main/riptide-failsafe#custom-executor